webpack.config.js

```diff
@@ -34,9 +35,12 @@ module.exports = {
   devServer: {
     port: 3000,
     open: true,
+    disableHostCheck: true,
     proxy: {
-      '/api': 'http://localhost:8080'
-    }
+      '/api/**': 'http://localhost:8080'
+    },
+    historyApiFallback: true,
+    public: 'http://v9_website.ngrok.io'
   },
   plugins: [
     new CleanWebpackPlugin([outputDirectory]),
```

Comment:
The auto indentation in this file is wrong I think, hard to tell on github

Reply:
Nope, github is just being difficult. ESLint would yell at me otherwise

Reply:
Why is there a diff here?

Reply:
I added the public line and changed the /api to be /api/**